src/com/android/launcher3/views/RecyclerViewFastScroller.java

```diff
@@ -58,7 +58,7 @@
 public class RecyclerViewFastScroller extends View {
     private static final String TAG = "RecyclerViewFastScroller";
     private static final boolean DEBUG = false;
-    private static final int FASTSCROLL_THRESHOLD_MILLIS = 40;
+    private static final int FASTSCROLL_THRESHOLD_MILLIS = 10;
```

Comment:
`ev.getEventTime() - mDownTimeStampMillis > FASTSCROLL_THRESHOLD_MILLIS` usually be false before, hard to trigger `mIsDragging`.

https://github.com/LawnchairLauncher/lawnchair/blob/0e8a2ce0a9024a52f1a0593a884de86e1f5fcd23/src/com/android/launcher3/views/RecyclerViewFastScroller.java#L275-L280

https://github.com/LawnchairLauncher/lawnchair/blob/0e8a2ce0a9024a52f1a0593a884de86e1f5fcd23/src/com/android/launcher3/views/RecyclerViewFastScroller.java#L308-L309